src/helpers/validator.spec.js

```diff
@@ -0,0 +1,72 @@
+const { validate, Symbols } = require('./validator');
+const chai = require('chai');
+
+chai.should();
+
+describe('validator', () => {
+  it('should return \'{ errors: [], payload }\' with empty validators array', async () => {
```

Comment:
Just a suggestion, you can use `"`:
`"should return '{ errors: [], payload }' with empty validators array"`